src/__experimental__/components/numeral-date/numeral-date.component.js

```diff
@@ -241,7 +248,7 @@ NumeralDate.propTypes = {
     const dateFormat = props[propName];
     const isAllowed =
       !dateFormat ||
-      OptionsHelper.dateFormats.find(
+      DATE_FORMATS.find(
```

Comment:
I think that `allowedDateFormats` would be a more descriptive name

Reply:
I'll rename this to ALLOWED_DATE_FORMATS